### PR TITLE
libbeat/scripts: make testsuite target re-runnable

### PR DESCRIPTION
**Pull request by 7AC**
Right now when you run `make testsuite` multiple times, it sometimes fails if `${BEAT_NAME}.template*.json` is not there. I added a `-f` to the `rm` command.

### CODE REVIEW

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -118,7 +118,7 @@ clean:: ## @build  Cleans up all files generated by the build steps
 	# Clean index pattern
 	rm -f $(PWD)/_meta/kibana/index-pattern/${BEAT_NAME}.json
 	# Remove temp templates
-	-rm ${BEAT_NAME}.template*.json
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: This should not be needed anymore on master as no templates are generated anymore. All are loaded dynamically from fields.yml.

File: libbeat/scripts/Makefile [hunk 2]
```diff
@@ -209,7 +209,8 @@ test: unit
 
 .PHONY: testsuite
 testsuite: ## @testing Runs all tests and generates the coverage reports
-testsuite: clean update
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: what is the difference here?

**Comment by ruflin** [comment]
It might be we try to fix a similar issue? https://github.com/elastic/beats/pull/3922

File: libbeat/scripts/Makefile (hunk 1 above)
Comment (7AC) [inline libbeat/scripts/Makefile]: Ok I'll remove it completely

File: libbeat/scripts/Makefile (hunk 2 above)
Comment (7AC) [inline libbeat/scripts/Makefile]: `clean` and `update` won't run in parallel, right now from what I can tell (on my laptop) they end up running in parallel and `update` tries to invoke `activate` from the `build` dir right after `clean` deleted all of it

File: libbeat/scripts/Makefile (hunk 1 above)
Comment (ruflin) [inline libbeat/scripts/Makefile]: I should have been more accurate here. It is still possible to generate the template through `make index-template` so removing can be still needed. But it (should) have no effect on the build as it is not used.

Comment (7AC) [inline libbeat/scripts/Makefile]: 👍 I'll keep it

**Comment by ruflin** [comment]
@7AC It would make sense to add somewhere a note that we don't support -j.

**Comment by 7AC** [comment]
@ruflin 👍 https://github.com/elastic/beats/pull/3937

**Pull request merged**